plugins/fastem_set_factory_values.py

```diff
@@ -50,10 +50,10 @@ def __init__(self, microscope, main_app):
             self.stage = main_data.stage
             self.mppc = main_data.mppc
             self.ccd = main_data.ccd
-            self._acquisition_tab = fastem_main_tab.acquisition_tab
+            self._acquisition_tab = fastem_main_tab.acquisition_tab.multi_beam_tab
```

Comment:
It becomes odd that the `multi_beam_tab` is called `self._acquisition_tab`.
Actually, this variable isn't used outside of the function. So I'd recommend to rename it just `multi_beam_tab`. 
(comment valid for both plugins).